subo/command/build.go

```diff
@@ -37,6 +38,15 @@ func BuildCmd() *cobra.Command {
 				return errors.New("🚫 no runnables found in current directory (no .runnable yaml files found)")
 			}
 
+			innerFiles, err := ioutil.ReadDir(bctx.Cwd)
+			if err != nil {
+				return errors.Wrapf(err, "Failed to list files in %s", bctx.Cwd)
+			}
+			fileName, _ := context.ContainsRunnableYaml(innerFiles)
```

Comment:
You can simplify this a bit by checking `if _, exists := context.ContainsRunnableYaml; exists { ... }`